ct-install.yaml

```diff
@@ -3,3 +3,4 @@ chart-dirs:
 validate-maintainers: false
 remote: origin
 target-branch: main
+helm-extra-args: --timeout 500s
```

Comment:
The backstage initContainer is taking to long for the default helm timeout so it had to be increased.